app/graphql/resolvers/surveys_search.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "search_object/plugin/graphql"
+require "graphql/query_resolver"
+
+class Resolvers::SurveysSearch
+  include SearchObject.module(:graphql)
+
+  # NOTE: following line is needed by the SearchObject module but cannot be used in `fetch_results`
+  scope { Survey::Poll.filtered_for_current_user(context[:current_user]) }
+
+  type types[Types::QueryTypes::SurveyPollType]
+
+  option :ids, type: types[types.ID]
+
+  # https://github.com/nettofarah/graphql-query-resolver
+
+  def fetch_results
+    # NOTE: scope from class is not available here, so we need to create a local scope here :/
+    scope = Survey::Poll.filtered_for_current_user(context[:current_user])
```

Comment:
diese Zeile kann also wieder weg, wenn das Model `Filter_by_role` integriert hat?

Reply:
nee, mit dem include wird es erst möglich, dass `filtered_for_current_user` aufgerufen werden kann

Reply:
haben wir bei allen anderen search Revolvern aber auch nicht so gemacht? ich steh echt auf dem Schlauch

Reply:
doch, guck in die searches von generic, news, tours, pois. wenn wir die rechteverwaltung anders machen, dann könnte sich das auch hier anders darstellen.

Reply:
wir die rechteverwaltung sich unterscheiden sodass hier ein anderer scope fällig wird?

Reply:
ist und bleibt genauso wie bei generic, news, tours, pois